tools/tool_united_heatmap.py

```diff
@@ -0,0 +1,123 @@
+import sys
+import pandas as pd
+import os
+import numpy as np
+import logging
+import seaborn as sns
+import matplotlib.pyplot as plt
+from matplotlib.patches import Patch
+import random
+
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger('main')
+
+if os.path.exists('/groups/pupko/orenavr2/'):
+    src_dir = '/groups/pupko/orenavr2/igomeProfilingPipeline/src'
+elif os.path.exists('/Users/Oren/Dropbox/Projects/'):
+    src_dir = '/Users/Oren/Dropbox/Projects/gershoni/src'
+else:
+    src_dir = '.'
+sys.path.insert(0, src_dir)
+
+from auxiliaries.pipeline_auxiliaries import load_table_to_dict
+
+
+def get_motifs_importance(biological_condition, bc_dir_path, rank_type, min_num_motifs, max_num_motifs, max_difference_from_last_motif, max_difference_from_fitst_motif):
+    important_motifs = []
+    motif_importance_path = os.path.join(bc_dir_path, f'{biological_condition}_{rank_type}_model','best_model','feature_importance.txt')
+    dict_importance = load_table_to_dict(motif_importance_path, 'Motif {} is not unique!!')
+    first_val = None
+    last_val = 0
+    for key, value in dict_importance.items():
+        if first_val == None:
+            first_val = float(value)
+            last_val = float(value)
```

Comment:
Avoid using same calls twice.
float casting is considered a heavy function.
Use  `last_val = first_val` instead